### PR TITLE
LPS-170229 Add test for convertToPageTemplateModal component

### PR DESCRIPTION
**Pull request by veroglez**
**Only review the last 2 commits**, the rest belong to this PR https://github.com/liferay-echo/liferay-portal/pull/10564
- https://github.com/liferay-echo/liferay-portal/commit/35e22874887533b0cfae196dc9505eb2b590a14b
- https://github.com/liferay-echo/liferay-portal/commit/a5eb02373a524d4bbdb8db7aac39c83feaeed748

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/plugins/convert-to-page-template-modal/components/ConvertToPageTemplateModal.test.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+);
+
+function renderConvertToPageTemplateModal() {
+	return render(
+		<StoreContextProvider
+			initialState={{
+				availableSegmentsExperiences: {},
+				segmentsExperienceId: '0',
+			}}
+		>
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/plugins/convert-to-page-template-modal/components/ConvertToPageTemplateModal.test.js:51]: We can use `StoreMother.Component` here (it is populated with a default state) :smile:

**Review by p2kmgcl — COMMENTED** [review]
This a tiny suggestion, the rest looks perfect :smile:

**Comment by p2kmgcl** [comment]
ci:stop

**Comment by veroglez** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by victorg1991** [comment]
resent here: https://github.com/liferay-echo/liferay-portal/pull/10630

**Pull request closed without merging**